common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -217,15 +217,15 @@ def test_add_students_to_cohort_failure(self):
 
         self.assertEqual(
             [
-                "0 students have been added to this cohort",
-                "1 student was already in the cohort"
+                "0 learners have been added to this cohort.",
+                "1 learner was already in the cohort"
```

Comment:
Right now the message would say:

"0 learners have been added to this cohort. 
- 1 learner was already in the cohort"

or, if they were batch scheduling a bunch of people at once, and some of them are preregistered:

"0 learners have been added to this cohort. # learners were preregistered to this cohort. These learners will automatically be added to the cohort when they enroll in the course.
- 1 learner was already in the cohort"

If there are more comments (like invalid email addresses or usernames), they'd be in the bulleted list, too.

@catong Do you think this is ok?